package-rfcx/rfcx/client.py

```diff
@@ -264,3 +264,25 @@ def streams(self, keyword=None, limit=1000, offset=0):
             List of streams"""
 
         return api_rfcx.streams(self.credentials.id_token, keyword, limit, offset)
+
+    def annotations(self, start=None, end=None, classifications=None, stream_id=None, limit=1000, offset=0):
+        """Retrieve a list of annotations
+
+        Args:
+            start: Minimum timestamp of the audio. If None then defaults to exactly 30 days ago.
+            end: Maximum timestamp of the audio. If None then defaults to now.
+            classifications: (optional, default= None) List of classification names.
+            stream_id: (optional,default= None) Limit results to a given stream.
```

Comment:
Let's change `stream_id` to be `stream` for any public functions exposed by the SDK.